app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/ActionGroup/AssertAdminCategoryGridPageDetailsActionGroup.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminCategoryGridPageDetailsActionGroup">
+        <arguments>
+            <argument name="category"/>
+            <argument name="imageName" type="string" defaultValue="magento"/>
+        </arguments>
+        <annotations>
+            <description>Assert category grid page basic columns values for a specific category</description>
+        </annotations>
+
+        <seeElement selector="{{AdminMediaGalleryCatalogUiCategoryGridSection.image(imageName)}}" stepKey="assertImageColumn"/>
+
+        <!--Name Column-->
+        <grabTextFrom selector="{{AdminMediaGalleryCatalogUiCategoryGridSection.columnValue('Name')}}" stepKey="grabNameColumnValue"/>
+        <assertEquals stepKey="assertNameColumn">
+            <expectedResult type="string">$$category.name$$</expectedResult>
+            <actualResult type="variable">grabNameColumnValue</actualResult>
+        </assertEquals>
+
+        <!--Path Column-->
+        <grabTextFrom selector="{{AdminMediaGalleryCatalogUiCategoryGridSection.columnValue('Path')}}" stepKey="grabPathColumnValue"/>
+        <assertStringContainsString stepKey="assertPathColumn">
+            <expectedResult type="string">$$category.name$$</expectedResult>
+            <actualResult type="variable">grabPathColumnValue</actualResult>
+        </assertStringContainsString>
+
+        <!--Products Column-->
+        <grabTextFrom selector="{{AdminMediaGalleryCatalogUiCategoryGridSection.columnValue('Products')}}" stepKey="grabProductsColumnValue"/>
+        <assertEquals stepKey="assertProductsColumn">
+            <expectedResult type="string">0</expectedResult>
```

Comment:
Let's move the hardcoded verification to a separate action group with a proper name as this verification does not really depend on the category provided for this action group